packages/plugins/users-permissions/server/controllers/auth.js

```diff
@@ -113,7 +113,7 @@ module.exports = ({ strapi }) => ({
           const cookieName = upSessions.cookie?.name || 'strapi_up_refresh';
           const cookieOptions = {
             httpOnly: true,
-            secure: Boolean(upSessions.cookie?.secure),
+            secure: process.env.NODE_ENV === 'production' || Boolean(upSessions.cookie?.secure),
```

Comment:
These don't seem right, because they will always resolve to `true` for production